### PR TITLE
added ticking casualties to battles

### PR DESCRIPTION
**Pull request by Wonobory**
added a text that shows the casualties taken of each side in the last tick, very useful for knowing if a battle is going well and how expensive in terms of manpower is being.

PR

### CODE REVIEW

**Comment by schombert** [comment]
you need to take out any windows-specific stuff you added for debugging

File: src/local_user_settings.hpp.bak [hunk 1]
```diff
@@ -0,0 +1 @@
+#pragma once
```
Comment (schombert) [inline src/local_user_settings.hpp.bak:1]: you seem to have added this file by accident

File: src/military/military.cpp [hunk 2]
```diff
@@ -5181,7 +5189,7 @@ void update_land_battles(sys::state& state) {
 					case unit_type::special:
 						state.world.land_battle_get_defender_support_lost(b) += str_damage;
 						break;
-					default:
```
Comment (schombert) [inline src/military/military.cpp:5184]: why did you remove the default case? yes, it shouldn't occur in a sane world, but if something really wacky has happened (like ship types appearing here), that is what it exists for. I also makes the static analysis tools happy

File: src/military/military.cpp [hunk 3]
```diff
@@ -5202,6 +5210,8 @@ void update_land_battles(sys::state& state) {
 				str_damage = std::min(str_damage, cstr);
 				state.world.regiment_get_pending_damage(att_front[i]) += str_damage;
 				cstr -= str_damage;
+				attacker_casualties += str_damage;
```
Comment (schombert) [inline src/military/military.cpp:5213]: you seem to be missing the equivalent addition for the backline inflicted defender casualties (there should be 2 of each, and you have only one place defender casualties are increases)

File: src/military/military.cpp (hunk 2 above)
Comment (Wonobory) [reply src/military/military.cpp:5184]: i think i removed it by accident

File: src/military/military.cpp (hunk 3 above)
Comment (Wonobory) [reply src/military/military.cpp:5213]: isn't already in line 5176?

Comment (schombert) [reply src/military/military.cpp:5213]: I see two attacker casualties += but only one defender casualties +=; there should be two of each (but I was just guessing about which was was dropped since I am just looking at the changes without much context here)

Comment (Wonobory) [reply src/military/military.cpp:5213]: well, idk if this is what you meant but i noticed that the front line damage dealt was flipped (so the attacker fronline damage dealt was added to attacker_casualties instead of defender_casualties)
